netci.groovy

```diff
@@ -87,7 +87,7 @@ commitPullList.each { isPr ->
     def myJob = job(jobName) {
       description("Windows CoreCLR unit tests")
             steps {
-              batchFile(""".\\build\\scripts\\cibuild.cmd ${(configuration == 'debug') ? '-debug' : '-release'} -testCoreClr""")
+              batchFile("""powershell -noprofile -executionPolicy RemoteSigned -file .\\build\\scripts\\cibuild.cmd ${(configuration == 'debug') ? '-debug' : '-release'} -restore -cibuild -bootstrap -coreClrBuild -testCoreClr -binaryLog""")
```

Comment:
This looks wrong. It's using powershell to target a CMD file. Not sure why we'd change this to use powershell here in any case.

Reply:
Yeah this should have been `build.ps1`. The problem is that `cibuild.cmd` passes the `-build` arg, which I made incompatible with `-coreClrBuild`. Would you prefer that both flags are required, and `coreClrBuild` just forces coreclr instead of desktop?

Reply:
I would do the following:

- Rename `-coreClrBuild` to `-buildCoreClr`.
- Allow mixing of `-build` and `-buildCoreClr`. Would actually set `-build` to true if `-buildCoreClr` is true in `Process-Arguments`

This would give `-buildCoreClr` a lot of symetry with `-buildAll`

Reply:
The reason I didn't do that is because, unlike buildAll, this doesn't build the same stuff as `-build` (it only builds Compilers.sln). Is that still OK?